### DISCUSSION TITLE
Domotizzare un monolocale

### DISCUSSION THREAD

**Discussion by FABIO-ANGELONI** [indomus/forum#2553] [GENERALE - Discussioni generiche]
Ciao ragazzi, sono un neofita del settore, volevo dei consigli per domotizzare il mio monolocale. Ho già pensato a qualcosa ma magari voi potete consigliarmi dispositivi migliori/più adatti.
Anzitutto poiché posseggo già un Raspberry pi3 sul quale avevo intenzione di installare HA e collegare a quest'ultimo un dongle USB per il protocollo zigbee.
Per quanto riguarda le luci avevo pensato di integrare all'interno delle 503 degli Shelley dimmer 2 per le classiche lampadine dimmerabili e degli Shelley rgbw2 per le strisce led.
Infine avevo pensato di acquistare vari sensori aqara per creare diverse automazioni e scenari. Potrebbe andare bene o avete qualcosa da consigliare/correggere?

**Comment by indomus** [comment]
Ciao Fabio,
in primis, installa Home Assistant e scopri cos'hai già in casa di integrabile, ti stupirà. Parti da [qui](https://indomus.it/formazione/home-assistant-su-raspberry-pi-varie-installazioni-le-cose-da-fare/).

In secondo luogo, hai già correttamente individuato cosa usare come attuatori, ovvero qualcosa di Wi-Fi - Shelly va più che bene.
Infine, prenditi qualcosa [tipo questo](https://indomus.it/componenti/itead-sonoff-zigbee-3-0-usb-dongle-plus-v2-zbdongle-e/) per pilotare tutti i sensori zigbee che nel tempo introdurrai in domotica.

**Reply by FABIO-ANGELONI** [reply]
grazie mille
